### PR TITLE
ddl:set the low priority for the adding the index

### PR DESCRIPTION
**Pull request by ciscoxll**
change parameters and fix the ci test problem

### CODE REVIEW

**Comment by shenli** [comment]
@ciscoxll Please add some benchmark results.

**Comment by ciscoxll** [comment]
Ok , positive test

File: store/tikv/kv.go [hunk 1]
```diff
@@ -253,8 +253,9 @@ func (s *tikvStore) BeginWithStartTS(startTS uint64) (kv.Transaction, error) {
 	return txn, nil
 }
 
-func (s *tikvStore) GetSnapshot(ver kv.Version) (kv.Snapshot, error) {
+func (s *tikvStore) GetSnapshot(ver kv.Version, priority int) (kv.Snapshot, error) {
```
Comment (shenli) [inline store/tikv/kv.go]: I think it is better to add an interface for kv.Snapshot such as `SetPriority()`.

Comment (ciscoxll) [reply store/tikv/kv.go]: already change the interface

File: store/tikv/kv.go [hunk 2]
```diff
@@ -259,6 +259,13 @@ func (s *tikvStore) GetSnapshot(ver kv.Version) (kv.Snapshot, error) {
 	return snapshot, nil
 }
 
+func (s *tikvStore) GetSnapshotSetPriority(ver kv.Version, priority int) (kv.Snapshot, error) {
+	snapshot := newTiKVSnapshot(s, ver)
```
Comment (shenli) [inline store/tikv/kv.go]: Call tikvStore.GetSnapshot()

File: kv/kv.go [hunk 3]
```diff
@@ -251,6 +251,8 @@ type Storage interface {
 	GetOracle() oracle.Oracle
 	// SupportDeleteRange gets the storage support delete range or not.
 	SupportDeleteRange() (supported bool)
+	// GetSnapshotSetPriority gets a snapshot set the priority
+	GetSnapshotSetPriority(ver Version, priority int) (Snapshot, error)
```
Comment (shenli) [inline kv/kv.go]: GetSnapshotWithPriority

**Comment by ciscoxll** [comment]
rebase branch
The new branch is here #5976

**Pull request closed without merging**